kolena/workflow/metrics/_plots.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import defaultdict
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Set
+
+from kolena._utils import log
+from kolena.workflow.evaluator import ConfusionMatrix
+from kolena.workflow.metrics._geometry import MulticlassInferenceMatches
+
+
+def compute_object_detection_test_case_confusion_matrix(
```

Comment:
Same comment as with the other pull request on naming to make things less verbose -- does the user need to think that that this is for `test_case` specifically? Any reason not to just have this as `compute_object_detection_confusion_matrix`?

On a side note, is `object_detection` necessary / should we split out OD metrics code into its own package? It feels as if it's implicitly assumed for other tools in `metrics`, but explicitly mentioned here.